dhall/dhall.cabal

```diff
@@ -706,3 +707,15 @@ Benchmark deep-nested-large-record
         dhall                                         ,
         gauge                     >= 0.2.3    && < 0.3
     Default-Language: Haskell2010
+
+Benchmark dhall-normalize
```

Comment:
If this is only a parser benchmark, I think it would be simpler to integrate it into the existing `dhall-parser` benchmarks.

Reply:
Done!